src/quiz/student.jsx

```diff
@@ -0,0 +1,30 @@
+export const Student = () => {
+  /* this is student info component */
+  const arrayStudent = [
```

Comment:
arrayStudent sounds like if it where an array of the type of students but instead it's the fields for student information, so we could call it studentFields maybe.